crates/oxc_linter/src/rules/react/jsx_props_no_spread_multi.rs

```diff
@@ -0,0 +1,100 @@
+use std::collections::HashMap;
+
+use oxc_ast::{ast::JSXAttributeItem, AstKind};
+use oxc_diagnostics::OxcDiagnostic;
+use oxc_macros::declare_oxc_lint;
+use oxc_span::{Atom, Span};
+
+use crate::{context::LintContext, rule::Rule, AstNode};
+
+fn jsx_props_no_spread_multi_diagnostic(span0: Span, span1: Span) -> OxcDiagnostic {
+    OxcDiagnostic::warn("Disallow JSX prop spreading the same identifier multiple times.")
```

Comment:
```suggestion
    OxcDiagnostic::warn(format!("Prop '{prop_name}' is spread multiple times."))
```

Reply:
I’ve revised the message.
https://github.com/oxc-project/oxc/pull/4866/commits/36f69a7a690d9af8f768619c2c7c1f06884eb118

Reply:
You added a help message. The error message should follow what's described in my comment. If you'd like to add a help message, make it at actionable sugggestion, e.g. "remove the first spread"